fastlane/lanes/release.rb

```diff
@@ -328,15 +328,16 @@ def check_pods_references
 end
 
 def trigger_buildkite_release_build(branch:, beta:)
-  environment = { BETA_RELEASE: beta.to_s }
+  environment = { BETA_RELEASE: beta }
   pipeline_file_name = 'release-build.yml'
 
   # When in CI, upload the release build pipeline inline in the current pipeline.
   # Otherwise, trigger a build via the Buildkite APIs.
   if is_ci
     buildkite_pipeline_upload(
       pipeline_file: File.join(PROJECT_ROOT_FOLDER, '.buildkite', pipeline_file_name),
-      environment: environment
+      # Both keys and values need to be passed as strings
+      environment: environment.to_h { |k, v| [k.to_s, v.to_s] }
```

Comment:
I'll try to port this to `release-toolkit` (and maybe Fastlane's `sh`, too) in the next few days. In the meantime, cc @iangmaia 

Also, TIL (via RuboCop) about `to_h` accepting a block. My solution was `.map { |k, v| [k.to_s, v.to_s] }.to_h` but this is clearly better.

Reply:
Ahh I'll probably have the same issue in Woo, which I'm tackling right now. Thanks for letting me know!